src/footer/js/footer.js

```diff
@@ -25,7 +25,7 @@ export default (data) => {
 
 	const $footer = $('footer',
 		{
-			class: `${prefix} ${data.hideMenus && 'skinny-footer'}`
+			class: `${prefix} ${data.hideMenus ? 'skinny-footer' : 'false'}`
```

Comment:
I would use `''` instead of `false` this is going to put the class `false` in the dom, right?

Reply:
@paulcpederson please review again..

Reply:
This will still put `false` in the dom, you should use an empty string `''` as I mentioned before...